packages/design-system/src/interfaces/forms.d.ts

```diff
@@ -102,6 +102,7 @@ export interface IInputProps
 
 export interface IInputFieldProps extends IInputProps {
   labelText?: string;
+  labelAdornment?: React.ReactNode;
```

Comment:
the same goes for IMultiSelectFieldProps, ITextAreaFieldProps, INumericInputFieldProps, ISelectFieldProps
lets create:
```
export interface IWithTextFieldProps {
  labelText?: string;
  id: string;
  inline?: boolean;
  description?: React.ReactNode;
  error?: string;
  labelAdornment?: React.ReactNode;
}
```
and extend mentioned interfaces with it

Reply:
Could you do me a favor?:D We need to also change one property in buttonGroup.d.ts - `size` should be optional.

Reply:
sure no problem :D 👍